enclave_manager/avalon_enclave_manager/kme/kme_enclave_manager.py

```diff
@@ -192,7 +193,12 @@ def PreProcessWorkOrder(self, **params):
             return wo_response_json["result"]
         else:
             logger.error("Could not preprocess work order at KME")
-            return wo_response_json
+            # For all negative cases, response should have an error field.
+            # Hence constructing JSONRPC error response with
+            # error code and message mapped to KME enclave response
+            err_code = wo_response_json["error"]["code"]
+            err_msg = wo_response_json["error"]["message"]
+            raise JSONRPCDispatchException(err_code, err_msg)
```

Comment:
Can we add a data field in `JSONRPCDispatchException` as "work order id"
https://github.com/hyperledger/avalon/blob/master/listener/avalon_listener/tcs_work_order_handler.py#L126. That way WPE will know for which work order ID KME returned error

Reply:
Done